bin/node-debug.js

```diff
@@ -194,7 +194,19 @@ function openBrowserAndPrintInfo() {
   );
 
   if (!config.options.cli) {
-    open(url);
+    // try to launch the URL in one of those browsers in the defined order
+    // (but if one of them is default browser, then it takes priority)
+    open(url, {
+        preferredBrowsers : ['chrome', 'chromium', 'opera']
+      }, function(err, okMsg) {
+        if (err) {
+           // unable to launch one of preferred browsers for some reason
+           console.log(err.message);
+           console.log('Please open the URL manually in Chrome/Chromium/Opera or similar browser');
+        } else if (okMsg) {
```

Comment:
In my code there are [4 cases of calling `cb` directly](https://github.com/jakub-g/biased-opener/blob/master/index.js#L83-149) which is either `cb(new Error)` or `cb(null, String)`, and one more is passing `cb` to `opener` which invokes [`child_process.execFile(cb: function (error:Error, stdout:Buffer, stderr:Buffer))`](http://nodejs.org/api/child_process.html#child_process_child_process_execfile_file_args_options_callback)

In my tests when I `console.dir(stdout)` in opener's callback it's always an empty string.

However it seems that success callback on Ubuntu (when going through `opener`, i.e. when opening default browser) is called **after the browser is closed** (on Windows it's after it's opened). When going through `browser-launcher`, the callback success is called always **immediately after the browser is opened**.

I may always use `browser-launcher` instead of using `opener` when opening the default browser, I was not sure which way to go actually. Any preferences about it?